app-text/texlive/texlive-2021.recipe

```diff
@@ -617,36 +820,53 @@ fixKSearchPaths()
 	sed -i -e "s@%TEXMFCNF.*@TEXMFCNF={$dataDir/texlive/texmf-dist/web2c,/boot/system/non-packaged/data/texmf-dist/web2c,/boot/home/config/settings/texmf-dist/web2c}@" $TEXMFCNF
 }
 
-BUILD()
+PATCH()
 {
 	# disable the install-bin-links target
 	sed -i -e 's/!WIN32/WIN32/g' am/bin_links.am
 
-	libtoolize --force --copy --install
-
-	cd texk/ptexenc
-	aclocal
-	automake --add-missing
-	cd -
-
-	autoreconf
-	mkdir -p ../build && cd ../build
+	./reautoconf
 
 	# may need to run this super early to ensure everything gets the right value...
 	fixKSearchPaths
+}
+
+BUILD()
+{
+	mkdir -p ../build && cd ../build
 
 	if [ $targetArchitecture = x86_gcc2 ] || [ $targetArchitecture = x86 ]; then
 		HOST=i586-pc-haiku
-	elif [ $targetArchitecture = x86_64 ]; then
-		HOST=x86_64-unknown-haiku
+	else
+		HOST=${targetArchitecture}-unknown-haiku
 	fi
 
-	# don't use secondary arch bindir
-	runConfigure --omit-buildspec --omit-dirs "binDir sbinDir" $sourceDir/configure \
-		--bindir=$prefix/bin --sbindir=$prefix/bin --disable-native-texlive-build --disable-cxx-runtime-hack \
-		--enable-shared --disable-static --with-banner-add="/Haiku" --with-editor='lpe +%d "%s"' --with-system-icu \
-		--without-cairo --with-system-freetype2 --with-system-libpng --with-system-zlib --without-x -C \
+	runConfigure --omit-buildspec --omit-dirs "binDir sbinDir" \
+		$sourceDir/configure \
+		--bindir=$prefix/bin --sbindir=$prefix/bin \
```

Comment:
This could be fixed as reported by @extrowerk 
```diff
- 		--bindir=$prefix/bin --sbindir=$prefix/bin \
+ 		--bindir=$commandBindDir --sbindir=$commandBindDir \